devito/ir/iet/nodes.py

```diff
@@ -1118,7 +1118,7 @@ class Callback(Call):
     errors occur during the creation of the call graph.
     """
 
-    def __init__(self, name, retval, param_types):
+    def __init__(self, name, retval, param_types, arguments=None):
```

Comment:
add `arguments` to docstring. 

What's `arguments` for? if it's for a call into the callback, why not just using a Call?